kotlin/generate-geodatabase/src/main/java/com/esri/arcgisruntime/sample/generategeodatabase/MainActivity.kt

```diff
@@ -0,0 +1,200 @@
+/* Copyright 2020 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.esri.arcgisruntime.sample.generategeodatabase
+
+import android.graphics.Color
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.Toast
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.app.AppCompatActivity
+import com.esri.arcgisruntime.concurrent.Job
+import com.esri.arcgisruntime.data.TileCache
+import com.esri.arcgisruntime.layers.ArcGISTiledLayer
+import com.esri.arcgisruntime.layers.FeatureLayer
+import com.esri.arcgisruntime.loadable.LoadStatus
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.mapping.Basemap
+import com.esri.arcgisruntime.mapping.view.Graphic
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol
+import com.esri.arcgisruntime.tasks.geodatabase.GenerateGeodatabaseJob
+import com.esri.arcgisruntime.tasks.geodatabase.GeodatabaseSyncTask
+import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.android.synthetic.main.dialog_layout.*
+
+class MainActivity : AppCompatActivity() {
+
+  private val TAG = MainActivity::class.java.simpleName
+  // define the local path where the geodatabase will be stored
+  private val localGeodatabasePath = cacheDir.path + getString(R.string.wildfire_geodatabase)
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_main)
+    // use local tile package for the base map
+    val sanFrancisco =
+      TileCache(getExternalFilesDir(null).toString() + getString(R.string.san_francisco_tpk))
+    val tiledLayer = ArcGISTiledLayer(sanFrancisco)
+
+    // add the map and graphics overlay to the map view
+    mapView.apply {
+      // create a map with the tile package basemap
+      this.map = ArcGISMap(Basemap(tiledLayer))
```

Comment:
```suggestion
      map = ArcGISMap(Basemap(tiledLayer))
```

this isn't needed here since theres no ArcGISMap called map